internal/utils/aggregator.go

```diff
@@ -86,6 +82,37 @@ func Aggregate_data(df dataframe.DataFrame) (dataframe.DataFrame, error) {
 }
 
 func filter_valid_csv_records(main_df dataframe.DataFrame) (dataframe.DataFrame, int) {
+
+	emptyWorkload := func() func(el series.Element) bool {
+		return func(el series.Element) bool {
+			if el.Type() == series.String {
+				if workload, ok := el.Val().(string); ok {
+					if workload == "<none>" || workload == "" {
```

Comment:
The main branch code accepts `""` and `"<none>" workloads. I like to know should we filter it out then?